README.md

```diff
@@ -1,5 +1,77 @@
+
+**Amazon_Vine_Analysis** [[GitHub](https://github.com/halmasieh/Amazon_Vine_Analysis)]:  Created an AWS RDS database with tables in pgAdmin. Picked a dataset from the Amazon Review datasets and extracted it. Determined bias of vine reviews. **Key Tech**: PySpark, SQL, Cloud Service (AWS)
+
+
+## Professional Experiences
+
+**Data Engineer, Jarvis (2021-present)**:
+- Implemented various data projects using tools and technologies such as Linux/Bash, RDBMS/SQL, Java, and Docker
+- Collaborated in the develop team and followed the Scrum Agile methodology by hosting daily meetings
+- used Git and GitFlow in every project
+
+**College Instructor, Vaughan College (2017-2021)**:
+- Planned and presented math lessons to facilitate students understanding in the college.
+- Prepared and distributed learning materials such as notes, assignments, tests, and final examinations
+- Ensured that the classroom remains safe and conducive to learning
+- Graded assessments in a timely manner
+- Documented and reported on students' progress
+- Attended meetings with parents and staff
+
+**Research Associate, Ryerson University (2015-2017)**:
+- Post-Doctoral Fellowship (PDF) at Electrical & Computer Department of Ryerson University
+- Designed a simulation of energy-efficient equipment and optimization of single house load management and energy generation
+- Worked on the Gray Predictive Models for using in HVAC residential
+
+**Assistant Professor, Isfahan University (2003-2015)**:
+- Faculty member of Isfahan University for more than 13 years
+- Developed curricula and delivered course materials
+- Researched with over 20 published papers with more than 528 citations
+- Supervised and advised on dissertation in undergraduate and graduate students
+- Specialized in analyzing and synthesizing applied problems and algorithms
+- Journal Reviewer of International Journals in Mathematics, Engineering, Economic and Science
+- Expertised in all areas of Applied Mathematics including but not limited to Numerical Analysis,  ODEs, PDEs, Finance Mathematical Modeling, Stochastic Calculus, Operational Research, Integral Equations, Linear Algebra, Numerical Integration, and Numerical Derivatives
+
+
+## Education
+**Azad University (2005-2010)**, Doctor of Philosophy of Applied Mathematics (Integral Equations), Mathematics
+
+**Yazd University (2001-2003)**, Master of Applied Mathematics (Optimal Control), Mathematics
+
+**Isfahan University (1997-2001)**, Bachelor of Applied Mathematics, Mathematics
+
+
+## Miscellaneous
+- Data Analytics- University of Toronto (2021)
+- A list of my publications can be accessed through [this link](https://github.com/halmasieh/Publications)
```

Comment:
A list of my publications can be accessed through https://github.com/halmasieh/Publications